packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -324,6 +324,29 @@ QUnit.test('The \'dataField\' option of a simple item should not affect existing
     assert.equal(form.getEditor('firstName').option('name'), 'UserName', 'Editor name is OK');
 });
 
+QUnit.test('slider labels should be rendered when used in forms (T1240185)', function(assert) {
```

Comment:
[test coverage]
This test check only behavior for RangeSlider. We need to test both Slider and RangeSlider. You can do it in a separate test but this test name should be updated as well
```suggestion
QUnit.test('rangeSlider labels should be rendered when used in forms (T1240185)', function(assert) {
```

Reply:
[e677538](https://github.com/DevExpress/DevExtreme/pull/27741/commits/e677538e7f9fe9f66641140a78596c41831a4359)